src/main/java/androidx/widget/SeekBar.kt

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2017 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.widget
+
+import android.widget.SeekBar
+
+/**
+ * Add an action which will be invoked when the SeekBar has started tracking touch.
+ *
+ * @return the [SeekBar.OnSeekBarChangeListener] added to the SeekBar
+ * @see SeekBar.OnSeekBarChangeListener.onStartTrackingTouch
+ */
+fun SeekBar.doOnStartTracking(action: (seekBar: SeekBar) -> Unit): SeekBar.OnSeekBarChangeListener =
+    OnSeekBarChangeListener(onStartTracking = action)
+
+/**
+ * Add an action which will be invoked when the SeekBar has stopped tracking touch.
+ *
+ * @return the [SeekBar.OnSeekBarChangeListener] added to the SeekBar
+ * @see SeekBar.OnSeekBarChangeListener.onStopTrackingTouch
+ */
+fun SeekBar.doOnStopTracking(action: (seekBar: SeekBar) -> Unit): SeekBar.OnSeekBarChangeListener =
+    OnSeekBarChangeListener(onStopTracking = action)
+
+/**
+ * Add an action which will be invoked when the SeekBar has the progress changed.
+ *
+ * @return the [SeekBar.OnSeekBarChangeListener] added to the SeekBar
+ * @see SeekBar.OnSeekBarChangeListener.onProgressChanged
+ */
+fun SeekBar.doOnChanged(action: (seekBar: SeekBar, progress: Int, fromUser: Boolean) -> Unit): SeekBar.OnSeekBarChangeListener =
+    OnSeekBarChangeListener(onChanged = action)
+
+/**
+ * Set the listener to this SeekBar using the provided actions.
+ */
+fun SeekBar.OnSeekBarChangeListener(
```

Comment:
A method name must start with a lower case. addListener() would be a better name (or addChangeListener).

Reply:
I agree (there is even a lint warning). I tried to follow the Animator (and others) convention, but SeekBar does not have an internal lowercased method.. It only allows to set the value, the other methods add to an ArrayList.

Reply:
You can call it `setOnSeekBarChangeListener` since it has a different signature.

Reply:
Fixed on latest commit (f133743). Thanks!